### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by vj-abishek**
#10

### CODE REVIEW

**Review by sabesansathananthan — COMMENTED** [review]
Nice

**Comment by vj-abishek** [comment]
Thanks

File: src/pages/LandingPage.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class Home extends Component {
     event.preventDefault();
 
     if (this.state.username.trim() !== "") {
-      this.props.history.push("/user/" + this.state.username + "/resume");
+      this.props.history.push("/user/" + this.state.username.toLowerCase() + "/resume");
```
Comment (sabesansathananthan) [inline src/pages/LandingPage.js:33]: Do all Github usernames have lower case letters?

File: src/pages/LandingPage.js [hunk 2]
```diff
@@ -47,10 +47,10 @@ class Home extends Component {
           style={{ flexDirection: "column" }}
         >
           <Typography variant="h5" align="center">
-            React Github Resume {packageJson.version}
+            Github Resume Generator {packageJson.version}
           </Typography>
           <Typography variant="body1" className="description" align="center">
-            Please enter Github profile name with case sensitive
+            Please enter Github username
```
Comment (sabesansathananthan) [inline src/pages/LandingPage.js:53]: GitHub

File: src/pages/LandingPage.js (hunk 1 above)
Comment (vj-abishek) [reply src/pages/LandingPage.js:33]: Sorry, thought that usernames are lowercase. Wait will change it

File: src/pages/LandingPage.js (hunk 2 above)
Comment (vj-abishek) [reply src/pages/LandingPage.js:53]: I think `name with case sensitive` is not necessary

**Comment by vj-abishek** [comment]
@sabesansathananthan need to do any other changes?

File: src/pages/LandingPage.js (hunk 1 above)
Comment (sabesansathananthan) [reply src/pages/LandingPage.js]: Github usernames are case insensitive. so no need of .toLowerCase() function
 
https://stackoverflow.com/questions/14712475/will-changing-my-github-username-to-snakecase-break-anything.

File: src/pages/LandingPage.js (hunk 2 above)
Comment (sabesansathananthan) [reply src/pages/LandingPage.js:53]: yes.. that's ok...  In GitHub ... H - Capital

Comment (vj-abishek) [reply src/pages/LandingPage.js:53]: Okay, wait

File: src/pages/LandingPage.js (hunk 1 above)
Comment (vj-abishek) [reply src/pages/LandingPage.js]: @sabesansathananthan okay

File: src/pages/LandingPage.js (hunk 2 above)
Comment (vj-abishek) [reply src/pages/LandingPage.js]: @sabesansathananthan need any other changes?

**Review by sabesansathananthan — COMMENTED** [review]
Thank you

**Pull request merged**